CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+## v2.11.0-rc.0 / 2019-07-03
+
+* [CHANGE] Remove max_retries from queue_config. #5649
```

Comment:
is it worth mentioning that it was removed because the value wasn't being used anymore? I guess giving the PR number is enough